### PR TITLE
653 Complete multi-user Devise setup

### LINKED ISSUE #653: Complete multi-user Devise setup
**Description**

Continue Devise advice for multi-user configuration from step 3, shown here:  https://github.com/heartcombo/devise/wiki/How-to-Setup-Multiple-Devise-User-Models#3-if-you-want-scoped-views

The first two steps are in the app, routes seem okay, but it's not possible to sign in as an AllCasaAdmin at http://localhost:3000/all_casa_admins/sign_in

Finish implementing multi-user model Devise until AllCasaAdmin can log in.

### PR DESCRIPTION
**Pull request by abachman**
### What github issue is this PR for, if any?

Resolves #653 

### What changed, and why?

- Extracted / ejected Devise controllers for User and AllCasaAdmin classes. 
- Set up DashboardController for AllCasaAdmin
- Added AllCasaAdmin can sign in and sign out tests
- dropped security meta test

### CODE REVIEW

File: app/controllers/all_casa_admins/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class AllCasaAdmins::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```
Comment (compwron) [inline app/controllers/all_casa_admins/confirmations_controller.rb:4]: messyyyy

File: app/views/all_casa_admins/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Update" %>
+  </div>
+<% end %>
+
+<h3>Cancel my account</h3>
+
+<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>
```
Comment (compwron) [inline app/views/all_casa_admins/registrations/edit.html.erb:41]: This seems odd for CASA

**Review by compwron — APPROVED** [review]
Progressssss! 📈💪

File: app/views/all_casa_admins/registrations/edit.html.erb (hunk 2 above)
Comment (abachman) [reply app/views/all_casa_admins/registrations/edit.html.erb:41]: oh lol, Devise defaults

**Pull request merged**